scripts/exregional_make_orog.sh

```diff
@@ -312,7 +312,7 @@ mv_vrfy "${raw_orog_fp_orig}" "${raw_orog_fp}"
 #
 #-----------------------------------------------------------------------
 #
-if [ "${CCPP_PHYS_SUITE}" = "FV3_HRRR" ]; then
+if [ "${CCPP_PHYS_SUITE}" = "FV3_HRRR_gf" ]; then
```

Comment:
Do this mean "FV3_HRRR" will retire and be replaced by "FV3_HRRR_gf"?

Reply:
This should be if [ "${CCPP_PHYS_SUITE}" = "FV3_HRRR" ] || if [ "${CCPP_PHYS_SUITE}" = "FV3_HRRR_gf" ];

Reply:
changed